pkg/globalstats/stats.go

```diff
@@ -49,45 +52,81 @@ func Incr(name string, tags ...stats.Tag) {
 	engine.Incr(name, tags...)
 }
 
+func Disable() {
+	mut.Lock()
+	defer mut.Unlock()
+
+	if flusherStop != nil {
+		close(flusherStop)
```

Comment:
should we set `flusherStop` to nil after closing the channel? seems like a good housekeeping effort.  Otherwise, we will get a panic if Diable is called twice (not that that would be expected).  Setting this package variable to nil would require passing in the stop chan as an argument to `flusher` as mentioned elsewhere.